samples/javascript_nodejs/51.cafe-bot/dialogs/dispatcher/interruptionDispatcher.js

```diff
@@ -37,23 +42,33 @@ module.exports = {
             // add dialogs
             this.addDialog(new WhatCanYouDoDialog());
             this.addDialog(new QnADialog(botConfig, userProfileAccessor));
+
+            // add recognizer
+            const luisConfig = botConfig.findServiceByNameOrId(LUIS_CONFIGURATION);
```

Comment:
>LUIS_CONFIGURATION [](start = 63, length = 18)

I'm not sure how you are dealing with the msbot decoration that occurs on LUIS services in the generated .bot configuration file.
For example, if your msbot name that was deployed is called "cafe", your corresponding LUIS name in the generated .bot configuration should be cafe_<my_luis_name>.

Reply:
I updated the readme to promote use of the CLI tools to get setup with the services and if you do, the names here (line 17) will match the instructions. Of course if you choose a different name (or need to pick up the LUIS app name from msbot), you will see an error on launch and need to update line 17.